bsmetadata/experiments/with_metadata.py

```diff
@@ -40,88 +42,119 @@ def get_dataloaders(tokenizer, args):
     #
     # In distributed training, the load_dataset function guarantees that only one local process can concurrently
     # download the dataset.
+    data_files = {}
+    if args.train_file is not None:
+        data_files["train"] = args.train_file
+    if args.validation_file is not None:
+        data_files["validation"] = args.validation_file
```

Comment:
I put this block up because you can also specify data_files to be considered in a hub repo using the new `datasets` library feature (currently on master).